resources/js/components/Containers/EdgeClusterContainer.vue

```diff
@@ -92,6 +92,7 @@ export default {
       try {
         await navigator.clipboard.writeText(text);
       } catch (err) {
+        console.log(`Failed to copy to clipboard: ${text}`);
```

Comment:
```suggestion
        console.warn(`Failed to copy to clipboard: ${text}`);
```

Reply:
If you like :). I only put it in at all because the copy doesn't work over HTTP and there's no way to see what would have been copied...